alby-nostr-wallet-connect/umbrel-app.yml

```
@@ -0,0 +1,32 @@
+manifestVersion: 1
+id: alby-nostr-wallet-connect
+name: Nostr Wallet Connect
+tagline: The power of the zaps at the tip of your fingers.
+icon: https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_icon.svg
+category: Finance
+version: "0.1.0"
+port: 58000
+description: >-
+  Connect your LND node over Nostr.
+
+  Zap directly from within supported apps like Damus & Amethyst.
+
+  Amethyst step-by-step guide: https://blog.getalby.com/native-zapping-in-amethyst
+
+  Damus (currently only on Testflight) video guide: https://streamable.com/bareih
+developer: Alby
+website: http://getalby.com/
+submitter: kiwiidb
+submission: https://github.com/getAlby/nostr-wallet-connect
+repo: https://github.com/getAlby/nostr-wallet-connect
+support: https://github.com/getAlby/nostr-wallet-connect/issues
+gallery:
+  - https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_1.png
+  - https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_2.png
+  - https://raw.githubusercontent.com/getAlby/umbrel-community-app-store/master/alby-nostr-wallet-connect/images/nwc_3.png
```

Thanks for these gallery images! I will upload these to https://github.com/getumbrel/umbrel-apps-gallery and now that the app will be on our app store, we can simply change this to:

```suggestion
gallery:
  - 1.png
  - 2.png
  - 3.png
```